clients/nexus-client/src/lib.rs

```diff
@@ -85,6 +88,26 @@ progenitor::generate_api!(
     }
 );
 
+impl IdOrdItem for types::RunningSagaInfo {
+    type Key<'a> = &'a Uuid;
+
+    fn key(&self) -> Self::Key<'_> {
+        &self.saga_id
+    }
+
+    id_upcast!();
+}
+
+impl IdOrdItem for types::HeldDbClaimInfo {
+    type Key<'a> = &'a u64;
```

Comment:
I think I'd make this just `u64` instead of `&u64`? (And maybe the same for `Uuid` above? They're both small and `Copy`.)